dart/generate-website-screenshot/lib/main.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+Future<void> start(final req, final res) async {
+  Map<String, dynamic> data = jsonDecode(req.payload);
+
+  // Checking if url Parameter is present in the payload
+  if (!data.containsKey("url")) {
+    throw Exception("Missing parameter url.");
```

Comment:
Please use `res.json()` to return a properly structured error response instead of throwing an error. Make sure to return after calling `res.json()`.